lynk_up_server/lynk_up_server/serializers.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+from .models import User, Friend, Group, Event
+
+class EventSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = Event
+    fields = ('id', 'group_id', 'title', 'date', 'time', 'address')
```

Comment:
Are we leaving in `address` in for the FE team? Sorry if this has already been discussed

Reply:
Yeah, we discussed with FE that we will include it as a 'just in case' so that is available for them to use if they need it at some point in the future.